Dockerfile

```
@@ -1,4 +1,4 @@
-FROM ruby:2.5-alpine
+FROM ruby:3.02-alpine
```

Is that a typo? On my machine, `docker pull` on that yields: 

> failed to solve with frontend dockerfile.v0: failed to create LLB definition: docker.io/library/ruby:3.02-alpine: not found

But this builds:

```suggestion
FROM ruby:3.0.2-alpine
```

---

That seems more correct.

---

Done in https://github.com/exercism/ruby-test-runner/pull/47